action.yml

```
@@ -47,17 +47,25 @@ inputs:
     required: false
   release_label_name:
     description: The name of the label used to describe the helm release
-    default: "release"
+    default: 'release'
+    required: false
+  values:
+    description: Extra values to pass to the helm action (comma-separated NAME=VAL pairs)
+    required: false
+  values_yaml:
+    description: YAML string with extra values to use in a helmfile deploy
     required: false
 outputs:
   webapp-url:
-    description: "Web Application url"
+    description: 'Web Application url'
```

revert formatting change

```suggestion
    description: "Web Application url"
```